internal/nostr/nostr.go

```diff
@@ -126,11 +115,23 @@ func NIP47Handler(c echo.Context) error {
 	}
 }
 
-func processRequest(ctx context.Context, requestData *NIP47Request) (*nostr.Event, int, error) {
-	relay, err := nostr.RelayConnect(ctx, requestData.RelayURL)
+func (svc *NostrService) getRelayConnection(ctx context.Context, customRelayURL string) (*nostr.Relay, bool, error) {
+	if customRelayURL != "" && customRelayURL != svc.config.DefaultRelayURL {
+		logrus.Infof("connecting to custom relay: %s...", customRelayURL)
```

Comment:
when you do logging always do JSON logging which makes it easier to search in the log files